### PR TITLE
Adding fix plus icon position

### PR DESCRIPTION
**Pull request by manuelrojas**
### <samp>🤖 Generated by Copilot at c67f3eb</samp>

This pull request enhances the block editor by improving the floating menu plugin and correcting a typo. It modifies `floating.plugin.ts` to adjust the menu position on document load, and `dot-block-editor.component.ts` to fix the placeholder text.

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
image  

https://user-images.githubusercontent.com/3438705/231610358-a6b51952-162c-497e-a475-37556f970aee.mov

### CODE REVIEW

File: core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts [hunk 1]
```diff
@@ -86,6 +86,13 @@ export class FloatingActionsView {
         this.command = command;
         this.key = key;
         this.tippyOptions = tippyOptions;
+
+        document.addEventListener('DOMContentLoaded', () => {
```
Comment (zJaaal) [inline core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts:90]: This event will be subscribed even if you close the block editor, remove the event listener on `destroy` method in line 196

Comment (fmontes) [reply core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts:90]: Yeah, we can remove the listener in the destroy method.

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by zJaaal — APPROVED** [review]

File: core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts [hunk 2]
```diff
@@ -86,6 +86,8 @@ export class FloatingActionsView {
         this.command = command;
         this.key = key;
         this.tippyOptions = tippyOptions;
+
+        document.addEventListener('DOMContentLoaded', () => this.setButtonPosition());
```
Comment (fmontes) [inline core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts:90]: Here you are referencing an anon arrow function, while in the remove you reference directly `this.setButtonPosition`, you need to make sure you reference both the same so the `removeEventListener` works.

```suggestion
        document.addEventListener('DOMContentLoaded', this.setButtonPosition);
```

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts (hunk 2 above)
Comment (rjvelazco) [inline core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts:90]: To achieve what Freddy mentioned, we can do this:

```suggestion
        document.addEventListener('DOMContentLoaded', this.setButtonPosition.bind(this));
```

**Review by rjvelazco — CHANGES_REQUESTED** [review]

**Pull request merged**